apps/emqx_management/test/emqx_auth_mnesia_migration_SUITE_data/v4.1.json

```diff
@@ -0,0 +1,48 @@
+{
+    "acl_mnesia": [
+        {
+            "action": "sub",
+            "allow": true,
+            "login": "emqx_c",
+            "topic": "Topic/A"
+        },
+        {
+            "action": "pub",
+            "allow": true,
+            "login": "emqx_c",
+            "topic": "Topic/A"
+        }
+    ],
+    "apps": [
+        {
+            "desc": "Application user",
+            "expired": "undefined",
+            "id": "admin",
+            "name": "Default",
+            "secret": "public",
+            "status": true
+        }
+    ],
+    "auth_clientid": [],
```

Comment:
We should support importing the authentication data exported from `emqx-auth-clientid` and `emqx-auth-username`, so we need to test version 4.0 at that time.

Reply:
The scope of the PR seem to grow. @tigercl What do you think about splitting this part to a separate PR?

Reply:
Couldn't agree more.. Let's merge it as soon as possible

Reply:
Great. I will follow-up tomorrow (hopefully) with another small fix for the `clientid` import.